### PR TITLE
Add theme classes to style the page a little

### PR DESCRIPTION
**Pull request by vegeorgi**
TP - http://teampulse.telerik.com/view#item/345603
TP - http://teampulse.telerik.com/view#item/345102

### CODE REVIEW

File: login/login-page.xml [hunk 1]
```diff
@@ -3,16 +3,47 @@
...
     </ActionBar>
 
-     <StackLayout>
-        <Button text="Log in with Facebook" tap="onLoginFacebookButtonTap"></Button>
-        <Button text="Log in with Google +" tap="onLoginGoogleButtonTap"></Button>
-        <Label text="or"></Label>
+     <StackLayout class="p-t-15">
+        <Button class="btn btn-outline text-left"  
+            tap="onLoginFacebookButtonTap">
+            <FormattedString>
```
Comment (ADjenkov) [inline login/login-page.xml:9]: We should probably follow the [docs](https://docs.nativescript.org/cookbook/formatted-string-ng)

File: login/login-page.xml [hunk 2]
```diff
@@ -3,16 +3,47 @@
...
+                <!-- 
+                    To see icons, you need to add font awesome to your project.
+                    Learn more about icon font and how it can be added in this documentation article:
+                    https://docs.nativescript.org/ui/icon-fonts
+                -->
+                <Span text="&#xf09a;" class="fa"></Span>
+                <Span  text="     Log in with Facebook"></Span>
```
Comment (ADjenkov) [inline login/login-page.xml]: I find this  uncomfortable. May be we should think for some other solution or leave it without 'padding'?
Try some magic property from the [API](https://docs.nativescript.org/api-reference/classes/_text_span_.span.html) ?

File: signup/signup-page.xml [hunk 3]
```diff
@@ -1,18 +1,50 @@
...
+                <Span text="&#xf09a;" class="fa"></Span>
+                <Span  text="     Sign up with Facebook"></Span>
+            </FormattedString>
+        </Button>
+
+        <Button class="btn btn-outline text-left" 
+            tap="onSignupGoogleButtonTap">
+            <FormattedString>
+                <Span text="&#xf0d5;" class="fa"></Span>
+                <Span  text="   Sign up with Google +"></Span>
```
Comment (ADjenkov) [inline signup/signup-page.xml]: 2 spaces ` <Span  text="`

File: signup/signup-page.xml [hunk 4]
```diff
@@ -1,18 +1,50 @@
...
+        <Button class="btn btn-outline text-left"  
+            tap="onSignupFacebookButtonTap">
+            <FormattedString>
+                <!-- 
+                    To see icons you need to add font awesome to your project.
+                    Learn more about icon font and how it can be added in this documentation article:
+                    https://docs.nativescript.org/ui/icon-fonts
+                -->
+                <Span text="&#xf09a;" class="fa"></Span>
+                <Span  text="     Sign up with Facebook"></Span>
```
Comment (ADjenkov) [inline signup/signup-page.xml]: 2 spaces `<Span  text="`

**Review by ADjenkov — CHANGES_REQUESTED** [review]

File: login/login-page.xml (hunk 1 above)
Comment (vegeorgi) [reply login/login-page.xml:9]: I have used the same syntax as in master detail templates  - [here](https://github.com/NativeScript/template-master-detail/blob/master/cars/cars-list-page.xml) and [here](https://github.com/NativeScript/template-master-detail-kinvey/blob/master/cars/cars-list-page.xml), in order to be consistent :)

File: login/login-page.xml (hunk 2 above)
Comment (vegeorgi) [reply login/login-page.xml]: The same as above - [here](https://github.com/NativeScript/template-master-detail-kinvey/blob/master/cars/cars-list-page.xml#L43)

I will look for something better in the API :)

File: signup/signup-page.xml (hunk 4 above)
Comment (vegeorgi) [reply signup/signup-page.xml]: 👍

File: signup/signup-page.xml (hunk 3 above)
Comment (vegeorgi) [reply signup/signup-page.xml]: 👍

File: login/login-page.xml (hunk 2 above)
Comment (vegeorgi) [reply login/login-page.xml]: There is no other way to have offset between two spans in FormattedString, except adding the spaces. If @EmilStoychev  is OK to remove the space between the icon and the text, i will remove it :)

**Review by ADjenkov — APPROVED** [review]

**Review by MartoYankov — APPROVED** [review]

**Pull request merged**